### ISSUE TITLE
Linux Snap segmentation fault

### ISSUE THREAD

**Issue by dimitarnestorov** [getsentry/sentry-electron#98]
There some code inside `init` that causes electron@2.0.5 to crash when built as a snap.
![image](https://user-images.githubusercontent.com/8790386/42901256-cad73260-8ad3-11e8-9e5b-32916a137089.png)

I've made an example repo: https://github.com/dimitarnestorov/sentry-segfault
Here's how to get to the screenshot:
```
git clone https://github.com/dimitarnestorov/sentry-segfault.git
cd sentry-segfault
yarn
yarn build
snap install dist/electron-quick-start-test_1.0.0_amd64.snap --dangerous
snap run electron-quick-start-test
```

**Comment by steverandy** [comment]
Does sentry have an option to disable the auto breadcrumb. Having the same issue and thinking disabling the auto breadcrumb as temporary solution.

**Comment by dimitarnestorov** [comment]
@steverandy I don't think so. My temporary solution is to disable Sentry on Linux using `process.platform`, but in case you distribute your app for a platform other than snap you might want to use something similar to https://github.com/bitwarden/jslib/blob/3429b57db42a3e1e9948b870bf24fcc02ebc8a99/src/electron/utils.ts#L21
I've opened an issue in the electron repo in case you want to follow it: https://github.com/electron/electron/issues/13767

**Comment by steverandy** [comment]
@dimitarnestorov I have not tried this yet, but perhaps may work. 
https://github.com/costlocker/desktop/commit/deac25cbf6ac78c32eddc8eb582d453d6be751b6

What do you think?

**Comment by dimitarnestorov** [comment]
I'm not sure I understand your question.

**Comment by steverandy** [comment]
Sorry, I misunderstood. Please ignore the reply above.
Now able to run the app by disabling sentry on the main process.
Thanks.
